### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by tie-rack**
* YourKit!
* Newrelic!
* Dependency version bumps!
* Uberjars!
* Cruft removal!

### CODE REVIEW

File: resources/leiningen/new/kraken_works/project.clj [hunk 1]
```diff
@@ -5,22 +5,27 @@
...
                   :exclusions [commons-codec commons-logging]]
-                 [ch.qos.logback/logback-classic "1.1.7"]
-                 [org.immutant/core "2.1.5"]
-                 [democracyworks/kehaar "0.8.1"]]
-  :plugins [[lein-immutant "2.1.0"]]
+                 [ch.qos.logback/logback-classic "1.2.2"]
+                 [org.immutant/core "2.1.6"]
+                 [democracyworks/kehaar "0.10.4"]]
+  :plugins [[lein-immutant "2.1.0"]
+            [com.carouselapps/jar-copier "0.3.0"]]
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_works/project.clj]: This should be `[com.pupeno/jar-copier "0.4.0"]` now. That fixes the "Coping" bug.

File: resources/leiningen/new/kraken_works/PROJECT@.service.template [hunk 2]
```diff
@@ -26,11 +24,11 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
 ExecStart=/bin/bash -c 'docker run --name ${CONTAINER} \
   --link rabbitmq:rabbitmq \
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_works/PROJECT@.service.template]: I think we should drop the `--link rabbitmq:rabbitmq` too (Docker links are deprecated) and replace it with `--dns $COREOS_PRIVATE_IPV4`, `--env RABBITMQ_PORT_5672_TCP_ADDR=rabbitmq.service.consul`, and `--env RABBITMQ_PORT_5672_TCP_PORT=5672`. That's what a typical service template for a live component does today anyway.

File: resources/leiningen/new/kraken_works/PROJECT@.service.template [hunk 3]
```diff
@@ -26,11 +24,11 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
...
-  --link wildfly:wildfly \
   --env FAKE_ENV_VAR="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/{{name}}/fake/env/var?raw)" \
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_works/PROJECT@.service.template]: Now that NewRelic is included in the template, should we replace this with `--env NEW_RELIC_LICENSE_KEY=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/new-relic/license-key?raw)`?

**Review by cap10morgan — CHANGES_REQUESTED** [review]
Much improved! Just a few little things to consider.

File: resources/leiningen/new/kraken_works/project.clj (hunk 1 above)
Comment (tie-rack) [reply resources/leiningen/new/kraken_works/project.clj]: My greatest contribution to open source! How could I forget?

File: resources/leiningen/new/kraken_works/PROJECT@.service.template (hunk 2 above)
Comment (tie-rack) [reply resources/leiningen/new/kraken_works/PROJECT@.service.template]: I noticed that while working on the http-api template. I'll get that fixed here.

File: resources/leiningen/new/kraken_works/PROJECT@.service.template (hunk 3 above)
Comment (tie-rack) [reply resources/leiningen/new/kraken_works/PROJECT@.service.template]: Yessir :)

**Review by cap10morgan — APPROVED** [review]
📦

**Pull request merged**